upgrade/v2.5/script/upgrade.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+# Set the ENV var for KDD backend
+export DATASTORE_TYPE=kubernetes
+
```

Comment:
Would we want to add `set -e` so that the script exits if there are any errors? maybe also using `-o pipefile` to catch any of the errors from the pipe'd commands.
^ I think these still work for /bin/sh